src/store/actions/authentication.ts

```diff
@@ -62,6 +63,25 @@ export const checkCredentials = ({ email, password }) => {
                 console.log('authentication error: ', error);
             });
 
+        let accesskey = await AsyncStorage.getItem('access_token');
+        console.log(accesskey);
+        let decoded = jwtDecode(accesskey);
```

Comment:
instead of decoded maybe, decodedAccessKey to be more specific about what's decoded